packages/wps-office-deb/wps-office-deb.pacscript

```diff
@@ -0,0 +1,10 @@
+pkgname="wps-office-deb"
+gives="wps-office"
+pkgver="11.1.0.11723.XA"
+source=("https://wdl1.pcfg.cache.wpscdn.com/wpsdl/wpsoffice/download/linux/11723/${gives}_${pkgver}_amd64.deb")
+pkgdesc="One of the Best Office Solutions for Linux"
+repology=("project: wps-office")
+depends=("cabextract" "libmspack0" "ttf-mscorefonts-installer" "bsdextrautils" "xdg-icon-resource")
```

Comment:
```suggestion
depends=("cabextract" "libmspack0" "ttf-mscorefonts-installer" "bsdextrautils" "xdg-utils")
```
Maybe just try use xdg-utils since it included `xdg-icon-resource`.